### DISCUSSION TITLE
Is this Project dead?

### DISCUSSION THREAD

**Discussion by chris-tian78** [celzero/rethink-app#329] [Q&A] [answered]
No updates, no issue / bug fixing... whats going on?

**Comment by ignoramous** [comment, marked as answer]
The lead developer had personal and health issues. He's back. 

We are undertaking a bit of a refactor considering the app went through rapid changes in Nov/Dec last year, and as a result the bugs got out of control. Trying to reign in that a bit with proper code organisation and state management: #282
